playbooks/kubernetes/app-resource-tuning.md

```diff
@@ -0,0 +1,111 @@
+Kubernetes has knobs that allow you to allocate cpu/memory for an app.
+
+These settings are configured in k8s manifests (Hokusai staging/production.yml), in each app's repo. Hokusai templates come with example values for these settings:

```

Comment:
I like the terminology "k8s manifests" here - will try and use it elsewhere, such as in hokusai and in other documentation